kolibri/content/api.py

```diff
@@ -204,9 +178,9 @@ def children_of_kind(channel_id=None, content=None, kind=None, **kwargs):
     :param kind: str
     :return: QuerySet of ContentMetadata
     """
-    return content.get_descendants(include_self=False).filter(kind=kind).using(channel_id)
+    return content.get_descendants(include_self=False).filter(kind__kind=kind).using(channel_id)
 
-def update_content_copy(file_object=None, content_copy=None):
+def update_content_copy(channel_id=None, file_object=None, content_copy_path=None):
```

Comment:
I'm still skeptical about the whole "content copy" thing (both the naming -- why is it a "copy"? -- and the use of a Django FileField). Let's take a look at that together.